HW_2/src/components/Profile.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import "./Profile.css";
+
+export default class Profile extends React.Component{
+	render(){
+		return (<>
+			<form className="form-inline" onSubmit={(e)=>{
+				e.preventDefault();
+				this.props.header(e.target.elements.userId.value);
```

Comment:
don't use code like this `e.target.elements.userId.value`. You should just save data from the input to state on change